Update trainings (#291)
acend/website-hugo: site/content/en/trainings/openshift4ops.md

  img3 : "images/trainings/ACEND_CodeD@2x.jpg"

main_text: "
With OpenShift 4, Red Hat promises a Kubernetes solution that boasts «automated operations» and other benefits for operations and development.\n\n 
This two-day training in German or English provides a practical and clearly understandable overview of the challenges in Day 2 operations. Our trainers come from the field and are experienced OpenShift Engineers. Delivery takes place online or on-site."

accordion:
    enable : true
    item:
        - title : "Outcomes"
          icon : "1"
          content : "<ul>
<li>Get to know OpenShift 4 architecture and components</li>
<li>Install your own OpenShift 4 cluster</li>

> <li>Mastering Day 2 operations tasks without interruption</li>

Reviewer: ```suggestion
<li>Mastering day 2-operations tasks without interruption</li>
```

<li>Deepen monitoring and troubleshooting know-how</li>
</ul>"

        - title : "Prerequisites"
          icon : "2"
          content : "<ul>
<li>Participants will need a laptop with direct access to the Internet</li>
<li>Additionally, local admin rights or access to a terminal (in the browser) are required</li>
<li>Participants should have basic Linux and Kubernetes knowledge</li>
</ul>"

        - title : "Target Audience"
          icon : "3"
          content : "<ul>

> <li>Our training is aimed at System Engineers, OpenShift Engineers and others who want to learn about the operational aspects of OpenShift 4</li>

Reviewer: Wollen wir hier wieder "alle"?

```suggestion
<li>Our training is aimed at system engineers, OpenShift engineers and everyhone who wants to learn about the operational aspects of OpenShift 4</li>
```

<li>Basic knowledge of container technologies as well as OpenShift or Kubernetes is a prerequisite</li>
</ul>"

        - title : "Conditions"
          icon : "4"
          content : "<ul>
<li>CHF 2200 / participant for 2 days</li>
<li>Group classes consisting of 8 to 24 participants</li>

> <li>Includes catering, documentation and an OpenShift cluster in the cloud</li>

Reviewer: ```suggestion
<li>Includes catering, documentation and an OpenShift cluster in the cloud for each participant</li>
```
